packages/store/src/operators/of-action.ts

```diff
@@ -85,7 +85,7 @@ export function ofActionErrored(...allowedTypes: any[]) {
  */
 export function ofActionExecuting(...allowedTypes: ActionType[]) {
   return (o: Observable<ActionContext>) => {
```

Comment:
I not happy if we use `(o: Observable<ActionContext>)` instead `(stream: Observable<ActionContext>)` (or any name)

Reply:
ok

Reply:
regarding score > 0 , with at least one true we consider it "executing"